src/api/c/chart.cpp

```diff
@@ -185,7 +185,7 @@ fg_err fg_render_chart(const fg_window pWindow, const fg_chart pChart,
     try {
         getChart(pChart)->render(getWindow(pWindow)->getID(),
                                  pX, pY, pWidth, pHeight,
-                                 IDENTITY);
+                                 IDENTITY, IDENTITY);
```

Comment:
Can you not use `opengl` namespace ? Maybe use something like `fgl` ? It'd be nice to have something unique internally. @umar456 what do you think ?